sydent/http/servlets/store_invite_servlet.py

```diff
@@ -79,29 +79,36 @@ def render_POST(self, request):
         tokenStore.storeEphemeralPublicKey(ephemeralPublicKeyBase64)
         tokenStore.storeToken(medium, address, roomId, sender, token)
 
+        # Variables to substitute in the template.
         substitutions = {}
+        # Include all arguments sent via the request.
         for k, v in args.items():
             if isinstance(v, string_types):
                 substitutions[k] = v
         substitutions["token"] = token
 
-        required = [
+        # Substitutions that the template requires, but are optional to provide
+        # to the API.
+        extra_substitutions = [
             'sender_display_name',
             'token',
             'room_name',
             'bracketed_room_name',
             'room_avatar_url',
-            'sender_display_name',
+            'sender_avatar_url',
```

Comment:
Oh this is an unrelated fix, but `send_display_name` was here twice, I'm fairly certain this is supposed to be `sender_avatar_url` as that's a [speced property](https://matrix.org/docs/spec/identity_service/latest#invitation-storage).